packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/FlipbookViewer.js

```diff
@@ -1,17 +1,78 @@
-import React from 'react'
+import React, { useEffect, useRef, useState } from 'react'
 import { Box } from 'grommet'
 import PropTypes from 'prop-types'
+
 import locationValidator from '../../helpers/locationValidator'
+import { placeholder } from '../SingleImageViewer/hooks/useSubjectImage'
+
+import SingleImageViewer from '../SingleImageViewer/SingleImageViewer.js'
+import SVGImage from '../SVGComponents/SVGImage'
+import SVGPanZoom from '../SVGComponents/SVGPanZoom'
+import FlipbookControls from './components'
+
+const FlipbookViewer = ({
+  defaultFrame,
+  defaultFrameSrc,
+  onReady,
+  naturalHeight,
+  naturalWidth,
+  subject
+}) => {
+  const subjectImage = useRef()
+  const [currentFrame, setCurrentFrame] = useState(defaultFrame)
```

Comment:
I originally set this state variable to `defaultFrame - 1` with the understanding that `metadata.default_frame`'s minimum value is 1. However, the SubjectViewerStore and MultiFrameViewer are built on the assumption that `subjectViewer.frame` can be initialized to zero. A follow-up PR that refactors SubjectViewerStore, MultiFrameViewer, and Flipbook will be needed to handle multi-image subjects that specify a default frame other than the first in the `locations` array.

Reply:
`subjectViewer.frame` is an array index, so the first frame is 0 (if the subject doesn’t override it.)

Reply:
https://github.com/zooniverse/front-end-monorepo/issues/3792

Reply:
I think fixing that one line in the subject viewer store will fix this.

Reply:
I think so too, I'll follow-up with a PR that's specifically for #3792